openedx/openedx-platform
# Remove the old course info page post launch DEPR-108/TNL-7061

## Description

This PR removes the old Course Info page that is no longer used.

It removes the Course Info page,  `enable_course_home_improvements` flag,  SASS that was written specifically for Course Info, the waffle flag (`course_experience.DISABLE_UNIFIED_COURSE_TAB`), `enable_course_home_improvements` flag, `info_sidebar_name` advanced setting, and `edx.course.home.course_update.toggled` event.

Please see:
* https://openedx.atlassian.net/browse/TNL-7061
* https://openedx.atlassian.net/browse/DEPR-108 (where this was deprecated & communicated out)
* https://openedx.atlassian.net/wiki/spaces/~151054611/pages/2442395936/TNL-7061 (active checklist of tasks from TNL-7061)
* Prior art: https://github.com/edx/edx-platform/pull/24676

Useful information to include:
- Which edX user roles will this change impact? 
"Course Author", "Developer", "Operator".

## Testing instructions

- Go to the Pages view of Studio for a course. Notice the old `Home` tab:
![image](https://user-images.githubusercontent.com/1985317/110242021-0423c200-7f22-11eb-948b-cc223c8c42b3.png)

- is no longer visible:
![image](https://user-images.githubusercontent.com/1985317/110242035-0dad2a00-7f22-11eb-9221-9ea776b78221.png)

- Test adding and removing new pages, this should work
- Make a new course, `Home` tab is not present
- Route `'^courses/{course_id}/info$'` no longer exists on LMS side
- - Test route exists on master by setting `course_experience.unified_course_tab`  to Force Off in django admin (NOTE: I cannot get this flag to actually work on master - need some help here)
- - Test route does not exist on branch
 


## Deadline

None

## Other information

Include anything else that will help reviewers and consumers understand the change.
- Any special concerns or limitations? For example: deprecations, migrations, security, or accessibility.
Deprecation, as outlined in DEPR-108
